nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/coremodel/SQLDocumentLive.java

```diff
@@ -338,6 +340,35 @@ public String getChangeToken() {
         }
     }
 
+    @Override
+    public boolean validateUserVisibleChangeToken(String userVisibleChangeToken) {
+        if (userVisibleChangeToken == null) {
+            return true;
+        }
+        if (session.isChangeTokenEnabled()) {
+            Long sysChangeToken = (Long) getPropertyValue(Model.MAIN_SYS_CHANGE_TOKEN_PROP);
+            Long changeToken = (Long) getPropertyValue(Model.MAIN_CHANGE_TOKEN_PROP);
+            return validateUserVisibleChangeToken(sysChangeToken, changeToken, userVisibleChangeToken);
+        } else {
+            Calendar modified;
+            try {
+                modified = (Calendar) getPropertyValue(DC_MODIFIED);
+            } catch (PropertyNotFoundException e) {
+                modified = null;
+            }
+            if (modified == null) {
+                return true;
+            }
+            String currentToken = String.valueOf(modified.getTimeInMillis());
+            return userVisibleChangeToken.equals(currentToken);
```

Comment:
No you're right, I'm missing the legacy `dc:modified`-based change token code for DBS. I'll add it.